Cargo.toml

```diff
@@ -32,132 +32,132 @@ parity-scale-codec = { version = "3.0.0", default-features = false }
-cumulus-relay-chain-rpc-interface       = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "release-polkadot-v1.7.0", default-features = false }
-pallet-collator-selection               = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "release-polkadot-v1.7.0", default-features = false }
-parachains-common                       = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "release-polkadot-v1.7.0", default-features = false }
-staging-parachain-info                  = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "release-polkadot-v1.7.0", default-features = false }
+cumulus-client-cli                      = { version = "0.8.0" }
+cumulus-client-collator                 = { version = "0.8.0" }
+cumulus-client-consensus-aura           = { version = "0.8.0" }
+cumulus-client-consensus-common         = { version = "0.8.0" }
+cumulus-client-consensus-proposer       = { version = "0.8.0" }
+cumulus-client-consensus-relay-chain    = { version = "0.8.0", default-features = false }
+cumulus-client-network                  = { version = "0.8.0", default-features = false }
+cumulus-client-service                  = { version = "0.8.0", default-features = false }
+cumulus-pallet-aura-ext                 = { version = "0.8.0", default-features = false }
+cumulus-pallet-parachain-system         = { version = "0.8.0", default-features = false }
+cumulus-pallet-session-benchmarking     = { version = "10.0.0", default-features = false }
+cumulus-pallet-xcm                      = { version = "0.8.0", default-features = false }
+cumulus-pallet-xcmp-queue               = { version = "0.8.0", default-features = false }
+cumulus-primitives-aura                 = { version = "0.8.0", default-features = false }
+cumulus-primitives-core                 = { version = "0.8.0", default-features = false }
+cumulus-primitives-parachain-inherent   = { version = "0.8.0", default-features = false }
+cumulus-primitives-utility              = { version = "0.8.0", default-features = false }
+cumulus-relay-chain-inprocess-interface = { version = "0.8.0", default-features = false }
+cumulus-relay-chain-interface           = { version = "0.8.0" }
+cumulus-relay-chain-minimal-node        = { version = "0.8.0", default-features = false }
+cumulus-relay-chain-rpc-interface       = { version = "0.8.0", default-features = false }
+pallet-collator-selection               = { version = "10.0.0", default-features = false }
+parachains-common                       = { version = "8.0.0", default-features = false }
+staging-parachain-info                  = { version = "0.8.0", default-features = false }
 
 # ORML
-orml-vesting = { git = "https://github.com/ajuna-network/open-runtime-module-library.git", branch = "ds/dependency-use", default-features = false }
+orml-vesting = { git = "https://github.com/ajuna-network/open-runtime-module-library.git", branch = "remove-nested-workspace", default-features = false }
```

Comment:
Depend on another branch that has all the orml crates, which I am gonna need for the benchmarking.